### ISSUE TITLE
Crashed when travelling

### ISSUE THREAD

**Issue by FelixGrafschmidt** [gwdevhub/GWToolboxpp#1225]
In the past few weeks, I have experienced an increasing number of crashes when fast travelling via the map. I cannot say for sure but I think it might be related to travelling to quickly after arriving in an area since there appears to be some syncing issues when reloading in.
I attached the most recent crash dump but I am happy to provide any additional information you might need.
[6.21-20240624-185851-546324-477412.dmp](https://github.com/user-attachments/files/16820117/6.21-20240624-185851-546324-477412.dmp)

**Comment by 3vcloud** [comment]
Update toolbox and try again, you’re not on the latest version

**Comment by FelixGrafschmidt** [comment]
I was a little confused since I compared the "last modified" date of my dll with the date of the last release and it appeared to match. Now I see that the dll in the latest release was changed 8 hours ago. Does this happen regularly? Unfortunately uploads like that do not trigger notifications for people who are watching this repo.

Anyways, thank you very much for your support!

**Comment by DubbleClick** [comment]
Toolbox has an auto-updater built in. You should not disable it.
![image](https://github.com/user-attachments/assets/7a744a4d-81c0-42bc-aa9c-6ceda2f60453)

**Comment by FelixGrafschmidt** [comment]
I am very sorry to be lingering on this point. I have always set any auto updating to active, since I am lazy and love automation. I suppose I had started GW before the update went live this morning.
But I am still a little concerned about the changed file in the release since the provided source code in the release will no longer match the distributed file, won't it?
I know that release automation can be a real pain and I don't know how the pipelines look for this repo, but it just seems odd to me to perform a new release by updating an existing one, from an archival perspective as well.

**Comment by 3vcloud** [comment]
Nothing has changed, we had some bug reports but we didn't have a valid PDB to debug them; the pdb uploaded with the release was the wrong one.

The only way to have a valid PDB is to compile the dll again and make sure we upload them both together, which is what I did this morning - I then fed back to the users having bug reports asking them to retry and send me the new crash dump.

Toolbox 6.22 was release on the 29th June

**Comment by FelixGrafschmidt** [comment]
Thank you for your response, that does indeed alleviate my worries.

Unfortunately I was already on the most recent version, as I thought (I had already adapted my travel favorites to the new ui). I tested a little more and the crash still occurs.

(I also realized that I uploaded the oldest crash report instead of the newest one at the beginning of this issue, I am very sorry about that. Here is the actual newest one)
[6.22-20240830-214551-36700-17412.dmp](https://github.com/user-attachments/files/16821548/6.22-20240830-214551-36700-17412.dmp)
